tripper/triplestore.py

```diff
@@ -160,6 +161,26 @@ def __init__(
             kwargs: Keyword arguments passed to the backend's __init__()
                 method.
 
+        Attributes:
+            backend_name: Name of backend.
+            base_iri: Assigned to the `base_iri` argument.
+            closed: Whether the triplestore is closed.
```

Comment:
What does this mean? can we have an example?

Reply:
These are class attributes. Documented using the google docstring style: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

It means that you can do the following

```python
>>> ts = Triplestore("rdflib")
>>> ts.closed
False
>>> ts.close()
>>> ts.closed
True

```